### DISCUSSION TITLE
Main menu question

### DISCUSSION THREAD

**Discussion by jewbmx** [Goldenfreddy0703/Otaku-Testing#12] [Q&A]
Did a fresh install of this newest version on my firestick to try testing it out and for some reason it wont show the movies and shows categories on the menu.
Out of the 11 menu items in the settings i can see all the others besides them 2 lol.
Not sure if thats a issue or just by design currently tho.

**Comment by Goldenfreddy0703** [comment]
Ok so funny story, i tried to add the movies and shows category but i also wanted to make the code shorter but i never really finished that so im probably just gonna remove it at the moment cause adding the movies and tv shows was a mess on the old Otaku so im not really sure how i could add it to the new Otaku.

**Comment by Goldenfreddy0703** [comment]
Hey @jewbmx , been awhile and have not looked in discussions for awhile, i added added Movies and TV Show menus and some more as well a few weeks ago. Hope this helps and glad i was able to make the code shorter.
Thanks!!!
